.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ jobs:
   # Make the Sphinx documentation for the project; without errors or warnings.
   make-docs:
     docker:
-      - image: circleci/python:3.6
+      - image: circleci/python:3.7
```

Comment:
Shouldn't we use 3.7.5 perhaps? Since that is what is the "latest" AND it is what the Dockers are using?

Reply:
I'll test this.